frontend/src/features/SidebarSuite/SidebarSuite.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useState } from "react";
+import { useResultPageType } from "@components/hooks/useResultPageType";
+import { useSettingsDrawer } from "@components/hooks/useSettingsDrawer";
+import { TabContext } from "@mui/lab/";
+import { Box, Drawer, Toolbar } from "@mui/material";
+
+import { TabContent } from "./TabContent";
+
+export function SidebarSuite() {
+  const { isSettingsOpen, setIsSettingsOpen, drawerWidth } =
+    useSettingsDrawer();
+  const [activeTab, setActiveTab] = useState("0");
+  const { isSearchPage } = useResultPageType();
+
+  const handleTabChange = useCallback(
+    (event: React.SyntheticEvent, newValue: string) => {
+      setActiveTab(newValue);
+    },
+    [setActiveTab],
+  );
+
+  return (
+    <Drawer
+      sx={{
+        width: drawerWidth,
+        flexShrink: 0,
+        "& .MuiDrawer-paper": {
+          width: drawerWidth,
+        },
+      }}
+      variant="persistent"
+      anchor="right"
+      open={isSettingsOpen}
+    >
+      <Toolbar />
+      <aside
+        id="db-results-settings-sidebar"
+        aria-label="settings and info tabs"
```

Comment:
needs i18n?

Reply:
can't recall if it was this particular case, but I know I deliberate set some labels as literals due to server mismatch issue. I think it's better to look at this as a dedicated issue after Next upgrade.